### ISSUE TITLE
Cant find flux_train.py file?

### ISSUE THREAD

**Issue by PheonixAi420** [kohya-ss/sd-scripts#1548]
I updated my Kohya looking forward to training a Flux lora but I cant find the flux_train.py file. Where is this located or can someone tell me how to do it? I'm not sure what is the issue.

**Comment by peteallen** [comment]
It's in the sd3 branch.

**Comment by PheonixAi420** [comment]
Yeah I had to download it manually and put the files in the folder because they weren't in the folder when I updated it. The sd3 were but not Flux. Now I have the problem of running the file. I try to run it and the terminal opens for a split second and closes. Tried running it in visual studio code and it gives the error of "no module toml" found although I checked and the toml file is in the appdata/local folder so I am confused to say the least. It seems there is a lot of issues with flux training at the moment.

**Comment by PheonixAi420** [comment]
I think one issue was that I didn't check the box "Add paths to environment" when installing python. For some reason the Flux scripts weren't added when I updated the program. So I went to the SD3 branch on here and downloaded the raw file and put it in the folder myself. I haven't tried it yet because I am waiting on a new GPU I ordered but hopefully I can get it working. I am not sure why the flux files weren't added, I even did a fresh install and no flux scripts were found.
